### PR TITLE
Add quiz: run a node

### PR DESCRIPTION
**Pull request by wackerow**
## Description
- Adds a six-question quiz for "run a node" topic

## Preview
https://ethereumorgwebsitedev01-quizrunanode.gatsbyjs.io/en/run-a-node#quiz

### CODE REVIEW

File: src/intl/en/learn-quizzes.json [hunk 1]
```diff
@@ -324,5 +324,52 @@
   "h005-c-label": "Eth1",
   "h005-c-explanation": "Eth1 was the original name given to the execution layer, not the consensus layer.",
   "h005-d-label": "Staking",
-  "h005-d-explanation": "Staking is depositing ETH into a smart contract to help secure the chain."
+  "h005-d-explanation": "Staking is depositing ETH into a smart contract to help secure the chain.",
+  "i001-prompt": "What is required to 'run a node'?",
```
Comment (minimalsm) [inline src/intl/en/learn-quizzes.json:328]:
```suggestion
  "i001-prompt": "What is required to run a node?",
```

File: src/intl/en/learn-quizzes.json [hunk 2]
```diff
@@ -324,5 +324,52 @@
...
+  "i001-a-label": "Run software, with modest hardware, while online",
```
Comment (minimalsm) [inline src/intl/en/learn-quizzes.json:329]: Wording here is weird. I'd drop 'while online' or change the wording somehow.

```suggestion
  "i001-a-label": "Running client software with modest hardware while staying online.",
```

File: src/intl/en/learn-quizzes.json [hunk 3]
```diff
@@ -324,5 +324,52 @@
...
+  "i001-b-label": "Deposit 32 ETH to earn rewards",
+  "i001-b-explanation": "This is a requirement for staking—the process of becoming an active participant in network consensus. This is not required to simply run a sovereign copy of the blockchain, which requires NO ETH.",
+  "i001-c-label": "Operate powerful ASIC mining machines to reach network consensus",
+  "i001-c-explanation": "Although Ethereum previously used mining with powerful computers to reach consensus, this process has been replaced entirely by staking. Neither mining in the past, nor staking currently, are required to simply operate a sovereign copy of the blockchain.",
+  "i001-d-label": "Work full time in blockchain infrastructure",
+  "i001-d-explanation": "Software tooling has continued to improve over time making running a node from home as a novice much more approachable. Working full-time in blockchain infrastructure is by no means a requirement to get involved.",
+  "i002-prompt": "How much ETH must be staked to operate a node?",
```
Comment (minimalsm) [inline src/intl/en/learn-quizzes.json:337]:
```suggestion
  "i002-prompt": "How much ETH do you need to stake to run a node?",
```

File: src/intl/en/learn-quizzes.json [hunk 4]
```diff
@@ -324,5 +324,52 @@
...
+  "i002-a-label": "0",
+  "i002-a-explanation": "Operating an Ethereum node does NOT require any ETH. In contrast to operating a staking validator as part of a node setup, anyone is free to run client software and sync their own sovereign copy of the blockchain—no ETH required.",
```
Comment (minimalsm) [inline src/intl/en/learn-quizzes.json:339]:
```suggestion
  "i002-a-explanation": "Operating an Ethereum node does not require any ETH. In contrast to operating a staking validator as part of a node setup, anyone is free to run client software and sync their own sovereign copy of the blockchain—no ETH required.",
```

File: src/intl/en/learn-quizzes.json [hunk 5]
```diff
@@ -324,5 +324,52 @@
...
+  "i002-b-label": "8",
+  "i002-c-label": "16",
+  "i002-d-label": "32",
+  "i002-d-explanation": "Operating an Ethereum node does NOT require any ETH. In contrast to the 32 ETH required to activate a staking validator that participates directly in network consensus, anyone is free to run client software and sync their own sovereign copy of the blockchain—no ETH required.",
+  "i003-prompt": "How does running an Ethereum node benefit you as a user?",
```
Comment (minimalsm) [inline src/intl/en/learn-quizzes.json:344]:
```suggestion
  "i003-prompt": "What benefits do you get from running your own node?",
```

File: src/intl/en/learn-quizzes.json [hunk 6]
```diff
@@ -324,5 +324,52 @@
...
+  "i003-c-label": "Privacy",
+  "i003-c-explanation": "This is a benefit to users, but is not the only one. Without your own node, simply looking up your account balances typically requires sending a list of your accounts from your wallet, attached to your IP address, to a third-party provider who then is being trusted to provide you with the correct information.",
+  "i003-d-label": "All of the above",
+  "i003-d-explanation": "Running a node gives you full control and sovereignty over the data you're relying on, allowing you to privately view and verify the contents of the chain, and effectively guarantee that any valid transactions are not censored.",
+  "i004-prompt": "What hard drive storage is required for an Ethereum node?",
+  "i004-a-label": "512 GB SSD",
+  "i004-a-explanation": "Currently, no client software is capable of storing the chain using only 512 GB",
+  "i004-b-label": "2 TB Rotating",
```
Comment (minimalsm) [inline src/intl/en/learn-quizzes.json:356]: Careful of combining these concepts. I'd separate type of hard disk and storage capacity

File: src/intl/en/learn-quizzes.json [hunk 7]
```diff
@@ -324,5 +324,52 @@
...
+  "i005-c-label": "The energy used looking for proof-of-work is wasted",
+  "i005-c-explanation": "Ethereum no longer uses proof-of-work, and this was never a requirement of all node operators. Being offline simply means your node is no longer in sync with the latest changes on the network, and can re-sync by returning online.",
+  "i005-d-label": "Chain data is removed, and re-syncing from scratch is required",
+  "i005-d-explanation": "Simply going offline does not typically delete any saved chain data. Connecting back to the internet will allow the software to resume where it left off to sync up with the latest transactions.",
+  "i006-prompt": "Running a node earns network rewards",
+  "i006-a-label": "True",
+  "i006-a-explanation": "Simply running client software to hold and verify your own copy of Ethereum does not earn you rewards. With a node set up, users can chose to also stake their ETH from home to participate in consensus and earn network rewards—this is not required to run a node, nor automatic.",
```
Comment (minimalsm) [inline src/intl/en/learn-quizzes.json:373]: a bit superfluous to the question 

```suggestion
  "i006-a-explanation": "Simply running client software does not earn you rewards. To earn rewards, you must also be staking.",
```

File: src/intl/en/learn-quizzes.json (hunk 5 above)
Comment (wackerow) [reply src/intl/en/learn-quizzes.json:344]: Yeah, was trying to contrast benefits to users vs the whole network, but didn't end up writing that question... I like this change

File: src/intl/en/learn-quizzes.json (hunk 6 above)
Comment (wackerow) [reply src/intl/en/learn-quizzes.json:356]: Two 🕊️ one 🪨? It's more just a chance to use a wrong answer as a teaching point.

**Review by corwintines — APPROVED** [review]

**Pull request merged**